
## 匹配相关参数
ICP:
    max_corr_dist : 1.2
    trans_eps : 0.01
    euc_fitness_eps : 0.36
    max_iter : 30
ICP_SVD:
    max_corr_dist : 0.5
    trans_eps : 0.001
    euc_fitness_eps : 0.36
    max_iter : 280
NDT:
    res : 1.0                 # volex  resolution
    step_size : 0.1    # 梯度下降的步长，越大下降越快，但是容易over shoot陷入局部最优
    trans_eps : 0.01    # 最大容差，一旦两次转换矩阵小于 trans_eps  退出迭代
    max_iter : 30         #   最大迭代次数
NDT_CPU:
    res : 0.8                 # volex  resolution
    step_size : 0.1    # 梯度下降的步长，越大下降越快，但是容易over shoot陷入局部最优
    trans_eps : 0.01    # 最大容差，一旦两次转换矩阵小于 trans_eps  退出迭代
    max_iter : 30         #   最大迭代次数
SICP:
    p : 1.0
    mu : 10.0
    alpha : 1.2
    max_mu : 1e5
    max_icp : 100
    max_outer : 100
    max_inner : 1
    stop : 1e-5


#$局部地图
key_frame_distance: 2.0 # 关键帧距离
local_frame_num: 20

#$当前帧
frame_filter: voxel_filter
#$局部地图
local_map_filter: voxel_filter
#$$滤波相关参数
voxel_filter:
    local_map:
        leaf_size: [0.6, 0.6, 0.6]
    frame:
        leaf_size: [1.3, 1.3, 1.3]

#$数据融合
kalman_filter_method: eskf # 选择卡尔曼滤波方法
fusion_strategy: pose_velocity
#$$状态误差卡尔曼相关参数
eskf:
    earth:
        # gravity can be calculated from https://www.sensorsone.com/local-gravity-calculator/ using latitude and height:
        gravity_magnitude: 9.91000
        # rotation speed, rad/s:
        rotation_speed: 7.292115e-5
        # 纬度:
        latitude:   48.9827703173
    # 协方差
    covariance:
        #先验状态的协方差
        prior:
            pos: 1.0e-6
            vel: 1.0e-6
            ori: 1.0e-6
            epsilon: 1.0e-6
            delta: 1.0e-6
        #过程噪声的协方差矩阵
        process:
            gyro: 1.0e-4
            accel: 2.5e-3
            bias_accel: 2.5e-3
            bias_gyro: 1.0e-4
        #测量噪声的协方差矩阵
        measurement:
            pose:
                pos: 1.0e-4
                ori: 1.0e-4
            pos: 1.0e-4
            vel: 2.5e-3
    motion_constraint:
        activated: true
        w_b_thresh: 0.13
    random_walk: 1          #是否使用随机游走，0：不使用（bias固定）,1：使用（bias随时间变化）